tests/env.py

```diff
@@ -0,0 +1,11 @@
+import sys
+import os
+
+# append module root directory to sys.path
+sys.path.append(
+    os.path.dirname(
```

Comment:
(I also made this change, it hasn't got updated because I removed the following line)